pilot/pkg/serviceregistry/kube/conversion.go

```diff
@@ -69,25 +65,12 @@ func convertLabels(obj meta_v1.ObjectMeta) model.Labels {
 	return out
 }
 
-// Extracts security option for given port from annotation. If there is no such
-// annotation, or the annotation value is not recognized, returns
-// meshconfig.AuthenticationPolicy_INHERIT
-func extractAuthenticationPolicy(port v1.ServicePort, obj meta_v1.ObjectMeta) meshconfig.AuthenticationPolicy {
-	if obj.Annotations == nil {
-		return meshconfig.AuthenticationPolicy_INHERIT
-	}
-	if val, ok := meshconfig.AuthenticationPolicy_value[obj.Annotations[portAuthenticationAnnotationKey(int(port.Port))]]; ok {
-		return meshconfig.AuthenticationPolicy(val)
-	}
-	return meshconfig.AuthenticationPolicy_INHERIT
-}
-
 func convertPort(port v1.ServicePort, obj meta_v1.ObjectMeta) *model.Port {
 	return &model.Port{
 		Name:                 port.Name,
 		Port:                 int(port.Port),
 		Protocol:             ConvertProtocol(port.Name, port.Protocol),
-		AuthenticationPolicy: extractAuthenticationPolicy(port, obj),
+		AuthenticationPolicy: meshconfig.AuthenticationPolicy_INHERIT,
```

Comment:
We can actually remove this AuthenticationPolicy from Port struct (model/service.go).